go.mod

```diff
@@ -0,0 +1,11 @@
+module github.com/google/docsy
+
+go 1.12
+
+//replace github.com/google/docsy/module v0.0.0-20220129185551-e4b42eecdf1d => e4b42eecdf1d
```

Comment:
Is this comment useful?

Reply:
No, that's an artifact and should be removed. I will  remove the line shortly.

Reply:
Removed in e444964a013e.